packages/styles/combobox.css

```diff
@@ -63,42 +32,6 @@
   padding: var(--space-smallest);
```

Comment:
Is this block still needed for other `input` in Combobox?

Reply:
Nope, removed!